README.rst

```diff
@@ -340,206 +342,202 @@ For example:
+                in the Cartridge :ref:`configuration file <cartridge-cli-overriding-default-options>`).
+
+The ``instances.yml`` file in the application directory contains parameters
+for starting Cartridge application instances. These parameters are parsed on
+:doc:`cartridge.cfg() </book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`
+call.
+
+Example ``instances.yml`` file:
+
+..  code-block:: yaml
+
+    myapp.router:
         advertise_uri: localhost:3301
         http_port: 8081
 
-      myapp.s1-master:
+    myapp.s1-master:
         advertise_uri: localhost:3302
         http_port: 8082
 
-  Parameters that can be specified in ``instances.yml`` are listed
-  `here <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`_.
-  The ``advertise_uri`` parameter is required.
+For the full list of parameters that can be specified in ``instances.yml``, read the
+:doc:`cartridge.cfg() documentation </book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`.
+``advertise_uri`` is a required parameter.
 
-  .. note::
+..  note::
 
-     The following parameters, if specified in ``instances.yml``, will be
-     overwritten by Cartridge CLI environment variables on
-     ``cartridge start``:
+    The following parameters, if specified in ``instances.yml``, will be
+    overwritten by Cartridge CLI environment variables on
+    ``cartridge start``:
 
-     * ``workdir``
-     * ``console_sock``
-     * ``pid_file``.
+    * ``workdir``
+    * ``console_sock``
+    * ``pid_file``.
 
-  You can also specify custom parameters in ``instances.yml``, but they
-  should be defined in the application code. The example below shows the usage
-  of the ``my_param`` custom parameter:
+You can specify custom parameters in ``instances.yml``, but they also
+have to be defined in your application code. See the following example, where
+``my_param`` is a custom parameter:
 
-  ``instances.yml``:
+``instances.yml``:
 
-  .. code-block:: yaml
+..  code-block:: yaml
 
-      myapp.router:
+    myapp.router:
         advertise_uri: localhost:3301
         http_port: 8081
         my_param: 'Hello, world'
 
-  ``init.lua``:
+``init.lua``:
 
-  .. code-block:: lua
+..  code-block:: lua
 
-     local argparse = require('cartridge.argparse')
-     local my_param = argparse.get_opts({my_param='string'})
+    local argparse = require('cartridge.argparse')
+    local my_param = argparse.get_opts({my_param='string'} )
 
-* ``--daemonize, -d`` starts the instance in background.
-  With this option, Tarantool also waits until the application's main script is
-  finished.
-  For example, it is useful if the ``init.lua`` requires time-consuming startup
-  from snapshot, and Tarantool waits for the startup to complete.
-  This is also useful if the application's main script generates errors, and
-  Tarantool can handle them.
+* ``--daemonize, -d`` starts the instance(s) in the background.
```

Comment:
You formatted some options into the table above, but left some (like this one) in a list. Now it is really hard to understand that all of them are the same list and could be passed to `cartridge start`.